christfellowshipchurch/web-app-v2
# Updated Share Button

### About
* Updated Share Button component to hide the text "Share" on mobile to provide more space and have it display on larger screens. 
* Also moved Share button on mobile to remove unnecessary space.

### Test Instructions
1. Go to https://web-app-v2-git-buttontext-christ-fellowship-church.vercel.app/studies/christfellowship-howtostudythebible
2. Go to inspect and inspect mobile. Button should only display logo. 
3. Compare with live version which can be found at https://www.christfellowship.church/studies/christfellowship-howtostudytheBible

### Screenshots
|Desktop|Mobile|
|---|---|
|<img width="1176" alt="Screenshot 2023-06-05 at 11 07 49 AM" src="https://github.com/christfellowshipchurch/web-app-v2/assets/93110503/bd84c2a7-2a6c-4df0-ae1d-432c1c17dcbd">|![image](https://github.com/christfellowshipchurch/web-app-v2/assets/46049974/492be48d-9aec-42b7-8abd-e4e1a3cbfe52)|

### Closes Tickets
[CFDP-2564](https://christfellowshipchurch.atlassian.net/browse/CFDP-2564)
[CFDP-2578](https://christfellowshipchurch.atlassian.net/browse/CFDP-2578)


[CFDP-2564]: https://christfellowshipchurch.atlassian.net/browse/CFDP-2564?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[CFDP-2578]: https://christfellowshipchurch.atlassian.net/browse/CFDP-2578?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
